ci/docker-compose.yml

```diff
@@ -49,6 +49,12 @@ services:
     image: public.ecr.aws/x5u3w5h6/rw-build-env:v20230208_05
     volumes:
       - ..:/risingwave
+  
+  rw-build-env-hdfs:
```

Comment:
Why do we need another image? It should be okay for us to integrate the hdfs environment in the default image, just like Kafka, etc.

Reply:
Currently our build image is about 2.7G, and adding `hadoop` environment will raise it to 3.7G, which is really heavy. After some offline discussion, we decide to build an image with `hadoop` only used in CI check, and do not include hdfs in e2e tests.

Reply:
IIUC, there were the steps of building the hdfs env in the `Dockerfile` on Feb.9 in this PR, so we get an image that can be used for hdfs tests by specifying this image tag here. However, this seems to prevent us from updating the image in the future like bumping the sqllogictest version by specifying a fixed image tag here. cc @xxchan

Reply:
>there were the steps of building the hdfs env in the `Dockerfile` on Feb.9 in this PR
Yes, that is to build the special hdfs image, and if we keep the steps of building the hdfs env in the `Dockerfile`, every time someone change this `Dockerfile`, it will build a new image with hdfs, but we only used in one step in pull request. 🤔